script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -452,6 +452,8 @@ Maybe the man noticed Arcueid coming for him,
 [sha:b5f68bc7b507992b1fd815a300d60b916aba21f8]{
 -- Page 305, Offset 17322.
 -- 正しくは、人間離れしたスピードで、この場から跳躍した。
+// not particularly big on "skitter", it's quite an insect-like word
+// for 跳躍. Perhaps "leaped"?
```

Comment:
I kinda like it because it _is_ more animal, creepy and the subject here is a Dead.